content/ember/v3/route-render-template.md

```diff
@@ -0,0 +1,137 @@
+---
+id: route-render-template
+title: Deprecate `Route#renderTemplate`
+until: '4.0.0'
+since: '3.26'
+---
+
+The `Route#render` and `Route#renderTemplate` APIs have been deprecated. These APIs are largely holdovers from a time where components where not as prominent in your typical Ember application and are no longer relevant. See [RFC #418](https://emberjs.github.io/rfcs/0418-deprecate-route-render-methods.html).
```

Comment:
The [Notion `deprecation` doc](https://www.notion.so/Deprecations-bbe769a13fb54bc8beafff175de4323a) only mentions `renderTemplate` but I assume we should also be adding a deprecation for `Route#render` per the RFC?

Reply:
Yes, correct.